basis/db/types/types-docs.factor

```diff
@@ -97,12 +97,12 @@ HELP: user-assigned-id-spec?
 
 HELP: bind#
 { $values
-     { "spec" "a SQL spec" } { "obj" object } }
+     { "spec" "an SQL spec" } { "obj" object } }
```

Comment:
If this is actually an ``sql-spec`` object, then we could say that instead of the text "an SQL spec".

But I haven't looked closely at it in awhile to remember if it is or if allows other things to be "SQL spec-like" to be passed in.